py/server/deephaven/table.py

```diff
@@ -1263,12 +1473,15 @@ def select_distinct(self, formulas: Union[str, Sequence[str]] = None) -> Table:
     #
     # region Filter
 
-    def where(self, filters: Union[str, Filter, Sequence[str], Sequence[Filter]] = None) -> Table:
+    def where(
+        self,
+        filters: Optional[Union[str, Filter, Sequence[str], Sequence[Filter]]] = None,
+    ) -> Table:
         """The where method creates a new table with only the rows meeting the filter criteria in the column(s) of
         the table.
 
         Args:
-            filters (Union[str, Filter, Sequence[str], Sequence[Filter]], optional): the filter condition
+            filters (Union[str, Filter, Sequence[str], Sequence[Filter]]): the filter condition
```

Comment:
optional.  I've already found 2 of these problems in this file.  The file needs to be checked very carefully.